src/VCreditCard.vue

```diff
@@ -223,4 +265,12 @@
         margin: 0;
         padding: 0 4px;
     }
+
+    .vcc-form-group input[data-error="true"], select[data-error="true"] {
```

Comment:
Use class names instead of data attributes, for example:
```html
<div class="vcc-form-group">
<input type="text" class="vcc-form-control is-invalid" />
</div>
```
```css
.vcc-form-control.is-invalid {
    border-color: red;
}
```